anesthetic/plot.py

```diff
@@ -216,7 +216,7 @@ def make_2d_axes(params, **kwargs):
     return fig, axes
 
 
-def grid_kde_plot_1d(ax, data, *args, **kwargs):
+def contour_plot_1d_grid_kde(ax, data, *args, **kwargs):
```

Comment:
"contour" only makes sense in the 2D case when it actually plots a contour plot, for 1D I'd leave it at `grid_kde_plot_1d` or flip it around for consistency `plot_1d_grid_kde`. Or `plot_plot_1d_grid_kde` if we go by the matplotlib names, but that sound stupid. Then I'd rather go for `line_plot_1d_grid_kde`.

Reply:
Changed to ``line_plot_*_kde`` in 3863a3a